utils/rpms/daos.spec

```
@@ -27,7 +27,7 @@ BuildRequires: python3-scons >= 2.4
 %else
 BuildRequires: scons >= 2.4
 %endif
-BuildRequires: libfabric-devel >= %{libfabric_version}
+BuildRequires: libfabric-devel < %{libfabric_version}
```

Since we won't be reverting this change as we'll need to bump the RPM version with any updates to the spec file, maybe we should do this to retain the current minimum version requirement:
```suggestion
BuildRequires: libfabric-devel >= %{libfabric_version} AND libfabric-devel < 1.18.0
```